okhttp/src/jvmMain/kotlin/okhttp3/internal/http/dates.kt

```diff
@@ -29,13 +29,11 @@ internal const val MAX_DATE = 253402300799999L
  * Most websites serve cookies in the blessed format. Eagerly create the parser to ensure such
  * cookies are on the fast path.
  */
-private val STANDARD_DATE_FORMAT = object : ThreadLocal<DateFormat>() {
-  override fun initialValue(): DateFormat {
-    // Date format specified by RFC 7231 section 7.1.1.1.
-    return SimpleDateFormat("EEE, dd MMM yyyy HH:mm:ss 'GMT'", Locale.US).apply {
-      isLenient = false
-      timeZone = UTC
-    }
+private val STANDARD_DATE_FORMAT by lazy {
```

Comment:
This looks like its removing the thread safety that ThreadLocal provided.

Reply:
Kotlin's lazy delegates are thread-safe by default.

Reply:
I believe its thread safe for construction, but you will then share an instance. I'm not clear how lazy replaces a ThreadLocal.